requirements/base.txt

```diff
@@ -6,13 +6,13 @@
 git+https://github.com/edx/XBlock.git@xblock-0.4.10#egg=XBlock==0.4.10
```

Comment:
You might as well bump this version at the same time (XBlock). Everyone forgets about poor little ora.